label_studio/io_storages/redis/serializers.py

```diff
@@ -40,3 +40,13 @@ def to_representation(self, instance):
     class Meta:
         model = RedisExportStorage
         fields = '__all__'
+
+    def validate(self, data):
+        data = super(RedisExportStorageSerializer, self).validate(data)
+
+        storage = RedisExportStorage(**data)
+        try:
+            storage.validate_connection()
+        except:
```

Comment:
please don't use so broad exceptions, better to write 
except Exception as exc: